constrain/Classes/BaseTypes.swift

```diff
@@ -91,8 +91,9 @@ public extension Constraints {
 // Height and width stuff
 public extension Constraints {
     /// Set the height constraint to a constant
+    // Can't have the constant param be anonymous or have a default or it would overload the anchor equivalent
     @discardableResult
-    func height(_ constant: CGFloat = 0.0, by relationship: Relationship = .equal, priority: UILayoutPriority = .required) -> Constraints {
+    func height(to constant: CGFloat, by relationship: Relationship = .equal, priority: UILayoutPriority = .required) -> Constraints {
```

Comment:
You should be able to omit the argument label, I think the bigger issue was that the anchor version had a default value of `nil` right?